### PR TITLE
Improve ram_group resource use common method

### PR DESCRIPTION
**Pull request by Lexsss**

### CODE REVIEW

File: alicloud/resource_alicloud_ram_group_test.go [hunk 1]
```diff
@@ -119,32 +117,147 @@ func testSweepRamGroups(region string) error {
...
+						"comments": "group comments",
+						"force":    "false",
+					}),
 				),
 			},
 		},
 	})
+}
+
+var ramGroupBasicMap = map[string]string{
```
Comment (ddaddaddda) [inline alicloud/resource_alicloud_ram_group_test.go]: name 去掉，加commit

File: alicloud/resource_alicloud_ram_group_test.go [hunk 2]
```diff
@@ -119,32 +117,147 @@ func testSweepRamGroups(region string) error {
...
+				),
+			},
+		},
+	})
+}
+
+func TestAccAlicloudRamGroup_multi(t *testing.T) {
+	var v *ram.GetGroupResponse
+	resourceId := "alicloud_ram_group.default.9"
+	ra := resourceAttrInit(resourceId, ramGroupMultiMap)
```
Comment (ddaddaddda) [inline alicloud/resource_alicloud_ram_group_test.go]: 177   ra := resourceAttrInit(resourceId, ramGroupMultiMap)  -> ra := resourceAttrInit(resourceId, nil)

File: alicloud/service_alicloud_ram.go [hunk 3]
```diff
@@ -311,3 +311,42 @@ func (s *RamService) WaitForRamAccessKey(id, useName string, status Status, time
...
+	})
+	if err != nil {
+		if RamEntityNotExist(err) {
+			return nil, WrapErrorf(err, NotFoundMsg, AlibabaCloudSdkGoERROR)
+		}
+		return nil, WrapErrorf(err, DefaultErrorMsg, id, request.GetActionName(), AlibabaCloudSdkGoERROR)
+	}
+	addDebug(request.GetActionName(), raw)
+	response, _ = raw.(*ram.GetGroupResponse)
+	return
```
Comment (ddaddaddda) [inline alicloud/service_alicloud_ram.go]: 添加类似   
if response  ... id != id {
return response, notFoundError
}

File: alicloud/resource_alicloud_ram_group.go [hunk 4]
```diff
@@ -48,11 +48,15 @@ func resourceAlicloudRamGroupCreate(d *schema.ResourceData, meta interface{}) er
...
+		return WrapErrorf(err, DefaultErrorMsg, "alicloud_ram_group", request.GetActionName(), AlibabaCloudSdkGoERROR)
 	}
 	addDebug(request.GetActionName(), raw)
 	response, _ := raw.(*ram.CreateGroupResponse)
 	d.SetId(response.Group.GroupName)
+	err = ramSercvice.WaitForRamGroup(d.Id(), Normal, DefaultTimeout)
+	if err != nil {
+		return WrapError(err)
+	}
 	return resourceAlicloudRamGroupUpdate(d, meta)
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_ram_group.go]: I think it should return read

File: alicloud/service_alicloud_ram.go [hunk 5]
```diff
@@ -715,3 +714,46 @@ func (s *RamService) WaitForRamRolePolicyAttachment(id string, status Status, ti
...
+		if err != nil {
+			if NotFoundError(err) {
+				if status == Deleted {
+					return nil
+				}
+			} else {
+				return WrapError(err)
+			}
+		}
+		if object.Group.GroupName == id {
```
Comment (xiaozhu36) [inline alicloud/service_alicloud_ram.go]: status != Deleted?

**Pull request merged**